### ISSUE TITLE
There is inconsistency in the campaign summary email_opened count and user timeline.

### ISSUE THREAD

**Issue by shahinraza1** [gophish/gophish#1937]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 0.90

Brief description of the issue:
**There is inconsistency in the campaign summary email_opened count and user timeline. I have sent a campaign to 50 users. Their timeline is showing that all the 50 emails are clicked at the same time, without opening it. But in the campaign summary, it is showing that Opened:50 and Clicked:50.**

What are you expecting to see happen? :
**All the count should be consistent in the timeline and summary.**
What are you seeing happen?
In summary email_opened: 50 in the timeline email doesn't open by any user.

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - 
 * Step 2 -

**Comment by glennzw** [comment]
There is a 'backfill' logic in the results summary (code logic [here](https://github.com/gophish/gophish/blob/master/static/js/src/app/campaign_results.js#L666)). Basically, it assumes that if the link has been clicked the email must have been read (and if data has been submitted, the link must have been clicked and the email must have been read).

Correct me if I'm wrong here, @jordan-wright

**Comment by shahinraza1** [comment]
Any users in this campaign don't even open or clicked the email. But still, it is showing that emails were clicked. Can you please let me know what might wrong happened in this.

**Comment by kurund** [comment]
@glennzw Users can disable showing of remote images. This will not trigger the open event and then they can click on the link. I feel we should not assume that clicking. == email opened.

**Comment by jordan-wright** [comment]
The logic around this is that they had to open the email to click the link. I’m not testing whether the email client loads external emails, I’m testing what the user is doing.

**Comment by kurund** [comment]
@jordan-wright Sorry for the delayed reply here. 

I agree if that's the logic then we should make it consistent with the timeline too. "Get Campaign Results" API does not show open true.

It would really help if "Get Campaign Summary" and "Get Campaign Results" are consistent with each other. That's not the case currently hence raised this issue.

Hope this helps.
